charts/core/src/Chart/hooks/useChart.ts

```diff
@@ -58,6 +59,12 @@ export function useChart({
     chartInstanceRef.current = chartInstance;
     chartInstance.setOption(chartOptions);
 
+    // Connects a chart to a group which allows for synchronized tooltips
+    if (groupId) {
+      chartInstance.group = groupId;
```

Comment:
Can there only be one `groupId` per chart?

Reply:
Yeah, there should be just one `groupId` per chart